app/config/parameters.yml.dist

```diff
@@ -36,9 +36,6 @@ parameters:
     sylius.currency_importer.open_exchange_rates: EDITME
     sylius.currency_importer.ecb.base_currency: EUR
 
-    paypal.express_checkout.username: EDITME
```

Comment:
It will crash only paypal checkout express but you are right i will figure out another solution.